src/order-history/actions.js

```diff
@@ -6,9 +6,9 @@ export const FETCH_ORDERS = new AsyncActionType('ORDER_HISTORY', 'FETCH_ORDERS')
 
 // FETCH ORDERS ACTIONS
 
-export const fetchOrders = (username, pageToFetch) => ({
```

Comment:
Thanks for the explanation on this - I am surprised we didn't get a ticket from support on users complaining about this.

Reply:
Me too. 😬

Reply:
They'd have to have more than 20 orders... so it may be pretty few and far between.